cluster/minio-tenant.go

```diff
@@ -29,8 +29,9 @@ func mkTenantMinioContainer(sgTenant *StorageGroupTenant, hostNum string) (v1.Co
 	envName := fmt.Sprintf("%s-env", sgTenant.ShortName)
 	volumeMounts := []v1.VolumeMount{}
 	tenantContainer := v1.Container{
-		Name:            fmt.Sprintf("%s-minio-%s", sgTenant.Tenant.ShortName, hostNum),
-		Image:           "minio/minio:latest",
+		Name:  fmt.Sprintf("%s-minio-%s", sgTenant.Tenant.ShortName, hostNum),
+		Image: "minio/minio:RELEASE.2019-10-12T01-39-57Z",
```

Comment:
because after this release, next version has breaking changes, so we want to make sure we upgrade when we are ready and we don't start seeing random results, we need to update madmin for example to be able to work with the next version of MinIO (i.e. minio:edge)

Reply:
there are breaking changes that's why it's better to stick to a version i.e tested and stable @Alevsk @dvaldivia - m3 can migrate to future releases as it gets tested etc and comes under m3's own release cycles.

Perhaps we can make this optional in future too, where users can choose their right version of MinIO